pkg/controller/ibmblockcsi/syncer/csi_controller.go

```diff
@@ -157,15 +155,64 @@ func (s *csiControllerSyncer) ensureContainersSpec() []corev1.Container {
 	}
 }
 
+func ensureDefaultResources() corev1.ResourceRequirements {
+	return ensureResources("50m", "100m", "50Mi", "100Mi")
```

Comment:
it looks like you put these limits\requests to all the containers of the node pod and controller pod.
Just wondering how you decide about the size here? is there any guideline that state these are the requested\limit for all the containers?  or it just a guess?

Another question, is it more standard to allow the user to set these params as part of the operator parameters? assuming the customer has issue with cpu and he need more, he will not be able to do anything.
do u think its good enough to keep it as this, or at least give the abiliy to configure it if user wants(lets say its a cluster with 200 nodes and with 1000 PVCs maybe 500m cpu is not enough its really loaded)?  thoughts?